internal/module/conditions_test.go

```diff
@@ -1,12 +1,11 @@
-package module_test
+package module
```

Comment:
Can we change the package name from module to conditions? I think module is a bit confusing here.